src/ray/raylet/monitor.cc

```diff
@@ -16,7 +16,7 @@ namespace raylet {
 Monitor::Monitor(boost::asio::io_service &io_service, const std::string &redis_address,
                  int redis_port)
     : gcs_client_(),
-      heartbeat_timeout_ms_(RayConfig::instance().num_heartbeats_timeout()),
+      heartbeat_timeout_ms_(RayConfig::instance().heartbeat_timeout_milliseconds()),
```

Comment:
Looking at the way the field is used, I actually think `num_heartbeats_timeout` is correct and that the field is misnamed.

Reply:
You are right, I fixed it!